app/scripts/modules/amazon/src/pipeline/stages/cloneServerGroup/awsCloneServerGroupStage.js

```diff
@@ -53,7 +53,7 @@ module.exports = angular.module('spinnaker.amazon.pipeline.stage.cloneServerGrou
     if (stage.isNew) {
       let useAmiBlockDeviceMappings = _.get($scope, 'application.attributes.providerSettings.aws.useAmiBlockDeviceMappings', false);
       stage.useAmiBlockDeviceMappings = useAmiBlockDeviceMappings;
-      stage.copySourceCustomBlockDeviceMappings = !useAmiBlockDeviceMappings;
```

Comment:
might as well remove the variable above and just assign the value of that `_.get` call, since we're not reusing it now.